packages/mperf/mperf.0.5.0/opam

```diff
@@ -0,0 +1,32 @@
+opam-version: "2.0"
+maintainer:   [ "Romain Calascibetta <romain.calascibetta@gmail.com>" ]
+authors:      [ "Vincent Bernardoff <vincent.bernardoff@ocamlpro.com>" "Pierre Chambart <pierre.chambart@ocamlpro.com>" ]
+homepage:     "http://github.com/dinosaure/bechamel"
+dev-repo:     "git://github.com/dinosaure/bechamel"
+bug-reports:  "http://github.com/dinosaure/bechamel/issues"
+license:      "GPL-2.0"
```

Comment:
Is this correct? The repo only mentions MIT. If it is correct is it `GPL-2.0-only` or the `or-later` version?

Reply:
Yes, I need to check the history because `mperf` was made by @chambart initially.

Reply:
Waiting on this to be resolved before merging.